src/tooltip/tooltip.spec.ts

```diff
@@ -292,6 +291,34 @@ describe('ngb-tooltip', () => {
         expect(windowEl.textContent.trim()).toBe('Great tip!');
       });
 
+      it('should accept placement in space seperated (second value should be applied)', () => {
```

Comment:
Let's open a separate feature PR for this later for ALL widgets supporting positioning